ki-shell/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jetbrains.kotlinx</groupId>
     <artifactId>ki</artifactId>
-    <version>0.4.5-SNAPSHOT</version>
+    <version>0.4.5</version>
```

Comment:
Please do not forget to increment version before the next release.

Reply:
BTW, thanks, there is no need to change versions manually, version release plugin does it for us